docs/what_combine_does/model_and_likelihood.md

```diff
@@ -0,0 +1,396 @@
+# Observation Models and Likelihoods
+
+## The Observation Model
+
+The observation model, $\mathcal{M}_0( \vec{\Phi}_{0})$ defines the probability for any set of observations given specific values of the input parameters of the model $\vec{\Phi}_0$.
+The probability for any observed data is denoted:
+
+$$ p_{\mathcal{M}_{0}}(\mathrm{data}; \vec{\Phi}_0 ) $$
+
+where the subscript $\mathcal{M}_{0}$ is given here to remind us that these are the probabilities according to this particular model (though usually we will omit it for brevity).
+
+Combine is designed for counting experiments, where the number of events with particular features are counted.
+The events can either be binned, as in histograms, or unbinned, where continuous values are stored for each event.
+The event counts are assumed to be of independent events, such as individual proton-proton collisions, which are not correlated with each other.
+
+The event-count portion of the model consists of a sum over different processes.  
+The expected observations, $\vec{\lambda}$, are then the sum of the expected observations for each of the processes, $\vec{\lambda} =\sum_{p} \vec{\lambda}_{p}$.
+
+The model can also be composed of multiple channels, in which case the expected observation is the set of all expected observations from the various channels $\vec{\lambda}_{0} = \{ \vec{\lambda}_{c1}, \vec{\lambda}_{c2}, .... \vec{\lambda}_{cN}\}$. 
```

Comment:
See above, we have _{0} here but in the previous paragraph, there's no subscript (prefer without)